src/clients/InsightsQueryClient.ts

```diff
@@ -534,6 +534,21 @@ class InsightsQueryClient {
     return response.data as Insights.InsightsQueryResponse<Insights.QueryGetFundingRateResponse>;
   }
 
+  async FundingHistoryGraphData(
+    query: Insights.QueryGetFundingHistoryGraphDataRequest
+  ): Promise<Insights.InsightsQueryResponse<Insights.QueryGetFundingHistoryGraphDataResponse>> {
+    const request = this.apiManager.path(
+      "market/funding/history/chart",
+      {},
+      {
+        market: query.market ?? "",
+        period: query.period ?? 0,
```

Comment:
what happens when `market: ""` and `period: 0` is passed in? does the response return anything? only allow optional params if when market /period is not specified then api returns something else e.g. full list 

else remove the optional flag for params and make it required